pkg/error/error_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 func TestGetErrorCount(t *testing.T) {
 	count := GetErrorCount([]ValidationErrors{})
-	if count != 0 {
+	if count != 3 {
```

Comment:
I don't get this change, nor anything that come next (about line and count) 

I mean how is it related to bumping version 🤔

Reply:
Yes, I agree. We should check what went wrong with the Release Please bot.

Reply:
Oh shit - I forgot to replace the `start` by `stop` in line 15